RELEASE-PROCESS.md

```diff
@@ -0,0 +1,94 @@
+This document describes the release process for NGINX Unit and as such is
+likely only of interest to NGINX Unit maintainers.
+
+# Create a preparatory branch
+
+You should create a new branch for doing this work. E.g.
+
+    $ git checkout -b x.y[.z]-prep master
+
+
+# Create a set of commits
+
+## unitctl
+
+Create a commit that updates the version of tools/unitctl. There are a
+few places where this needs updating, find them with
```

Comment:
Can we specify the exact places here?
I can predict without exact clarity a new team member may miss something.
Additionally, the version string exists in Cargo.lock multiple times.

Unless I am wrong this specifically is the following (from the unitctl directory)
- unitctl/Cargo.toml
- openapi-config.json
- unit-openapi/Cargo.toml
- unit-openapi/README.md (autogenerated?)
- unit-client-rs/Cargo.toml
- three separate places in Cargo.lock (can be updated by building unitctl after above changes)

Reply:
I've shown _exactly_ how to find them...

That list may be right today it may not be tomorrow...

Reply:
fair enough!